cmd/seid/cmd/root.go

```diff
@@ -342,15 +357,75 @@ func initAppConfig() (string, interface{}) {
 			LruSize:       1,
 			QueryGasLimit: 300000,
 		},
+		JSONRPC: *servercfg.DefaultJSONRPCConfig(),
 	}
 
-	customAppTemplate := serverconfig.DefaultConfigTemplate + `
+	customAppTemplate := sdkcfg.DefaultConfigTemplate + `
 [wasm]
 # This is the maximum sdk gas (wasm and storage) that we allow for any x/wasm "smart" queries
 query_gas_limit = 300000
 # This is the number of wasm vm instances we keep cached in memory for speed-up
 # Warning: this is currently unstable and may lead to crashes, best to keep for 0 unless testing locally
-lru_size = 0`
+lru_size = 0
+
+###############################################################################
+###                           JSON RPC Configuration                        ###
+###############################################################################
```

Comment:
why do we put this in the root.go instead of the params store (from genesis.json)

Reply:
I'm not sure what you mean? Do you mind clarifying?